features/ccx-upgrades-inference/perform_inference.feature

```diff
@@ -90,3 +90,336 @@ Feature: Upgrade Risks Prediction inference - test well known values
               }
             }
           """
+
+  Scenario: Check Inference Service does not predict risk for available operator conditions
+    Given The CCX Inference Service is running on port 8000
+     When I request the upgrade-risks-prediction endpoint in localhost:8000 with JSON
+          """
+            {
+              "alerts": [],
+              "operator_conditions": [
+                {
+                  "name": "authentication", 
+                  "condition": "Available", 
+                  "reason": "AsExpected"
+                }
+              ]
+            }
+          """
+     Then The status code of the response is 200
+      And The body of the response has the following schema
+          """
+            {
+              "required": [
+                "upgrade_risks_predictors"
+              ],
+              "type": "object",
+              "properties": {
+                "upgrade_risks_predictors": {
+                  "title": "Upgrade Risks Predictors",
+                  "type": "object",
+                  "properties": {
+                    "alerts": {
+                      "title": "Alerts",
+                      "type": "array"
+                    },
+                    "operator_conditions": {
+                      "title": "Degraded Operator Condition",
+                      "type": "array"
+                    }
+                  }
+                }
+              }
+            }
+          """
+      And The body of the response is the following
+          """
+            {
+              "upgrade_risks_predictors": {
+                "alerts":[],
+                "operator_conditions":[]
+              }
+            }
+          """
+
+  Scenario: Check Inference Service does not predict risk for less than 2 alerts
+    Given The CCX Inference Service is running on port 8000
+     When I request the upgrade-risks-prediction endpoint in localhost:8000 with JSON
+          """
+            {
+              "alerts": [
+                {
+                  "name": "APIRemovedInNextEUSReleaseInUse",
+                  "namespace": "openshift-kube-apiserver",
+                  "severity": "info"
```

Comment:
Please use "critical" so that we check the [inference filter](https://gitlab.cee.redhat.com/ccx/ccx-upgrades-inference/-/blob/main/ccx_upgrades_inference/inference.py#L37) and the [data-eng counting](https://gitlab.cee.redhat.com/ccx/ccx-upgrades-data-eng/-/blob/main/ccx_upgrades_data_eng/inference.py#L57)

Reply:
well, nvm, I've just seen it's tested later